### PR TITLE
Fix part of #1287: Fix lint issues in app module in the kotlin files  (settings.profile)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1287
`settings.profile` folder `ktlint` check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/settings/profile/ProfileEditActivity.kt [hunk 1]
```diff
@@ -10,7 +10,8 @@ const val KEY_PROFILE_EDIT_PROFILE_ID = "KEY_PROFILE_EDIT_PROFILE_ID"
 
 /** Activity that allows user to edit a profile. */
 class ProfileEditActivity : InjectableAppCompatActivity() {
-  @Inject lateinit var profileEditActivityPresenter: ProfileEditActivityPresenter
+  @Inject
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/settings/profile/ProfileEditActivity.kt:13]: Bring this to single line.

File: app/src/main/java/org/oppia/app/settings/profile/ProfileListActivity.kt [hunk 2]
```diff
@@ -8,7 +8,8 @@ import javax.inject.Inject
 
 /** Activity that allows users to select a profile to edit from settings. */
 class ProfileListActivity : InjectableAppCompatActivity() {
-  @Inject lateinit var profileListActivityPresenter: ProfileListActivityPresenter
+  @Inject
+  lateinit var profileListActivityPresenter: ProfileListActivityPresenter
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/settings/profile/ProfileListActivity.kt:12]: Ditto here and elsewhere.

File: app/src/main/java/org/oppia/app/settings/profile/ProfileEditActivity.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/settings/profile/ProfileEditActivity.kt:13]: Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

File: app/src/main/java/org/oppia/app/settings/profile/ProfileListActivity.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/settings/profile/ProfileListActivity.kt:12]: Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting

**Review by veena14cs — APPROVED** [review]
LGTM.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by anandwana001** [comment]
> LGTM, thanks.

Thanks for the review :)

**Pull request merged**